geps/gep-1709.md

```diff
@@ -355,9 +410,12 @@ profiles:
 > regarding conformance, e.t.c.). Optionally, it can be an email address or
 > a support URL (e.g. Github new issue page).
 
-The above report describes an implemenation that just released `v1` and has
-`Core` support for `TCP` functionality and fully supports both `Core` and
-`Extended` `HTTP` functionality.
+The above report describes an implementation that just released `v1`, uses gateway
+API `v0.8.0` `standard` channel, and has:
+
+* `HTTP` `core` and `extended` support in `standard` mode;
+* `HTTP` `core` and `extended` support in `enhanced` mode;
+* `TCP` `Core` support in `enhanced` mode.
```

Comment:
From the perspective of this document, I don't feel like `enhanced` has enough meaning or context here.

Reply:
Do you have other words in mind that could suit better our needs here?

Reply:
I put a note about this above, but if I'm right and the implementation can define the modes, then maybe a note here as well that says something like "remember that the mode values are arbitrary and chosen by the implementation" or something.

We could also make that clearer by making the names more clearly arbitrary - like calling the two modes `plain` and `with-the-lot` or something silly.

Reply:
> I put a note about this above, but if I'm right and the implementation can define the modes, then maybe a note here as well that says something like "remember that the mode values are arbitrary and chosen by the implementation" or something.

I agree that we need to specify this aspect; I added it [here](https://github.com/kubernetes-sigs/gateway-api/blob/342075e387f4b6534921be5b172dad734b705e7b/geps/gep-1709.md#implementation-mode).

as suggested by Nick, I changed the modes to `plain` and `with-the-lot`, to better highlight this aspect.